.github/workflows/ccpp.yml

```diff
@@ -0,0 +1,27 @@
+name: Radare2 CI
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version: ['12']
+        os: [ubuntu-latest, windows-latest, macOS-latest]
```

Comment:
Remove `windows-latest` and this will probably work, build steps are different for windows